release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+
+[id="ocp-4-7-18"]
+=== RHBA-2021:2502 - {product-title} 4.7.18 bug fix update
+
+Issued: 2021-06-28
+
+{product-title} release 4.7.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2502[RHBA-2021:2502] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:R2503[RHBA-2021:2503] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6139672[{product-title} 4.7.18 container image list]
+
+[id="ocp-4-7-18-features"]
+==== Features
+
+[id="ocp-4-7-18-builds-telemetry-metric"]
+===== New Telemetry metric for number of builds by strategy
+In this update, Telemetry includes a new `openshift:build_by_strategy:sum` gauge metric, which sends the number of builds by strategy type to the Telemeter Client. This metric gives site reliability engineers (SREs) and product managers visibility into the kinds of builds that run on {product-title} clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969963[*BZ#196963*])
+
+[id="ocp-4-7-18-bug-fixes"]
+==== Bug fixes
+
+* Previously, if you created a pipeline with parameters that had empty strings (`”`) as the default value, the fields in the {product-title} web console did not accept the empty strings. The current release fixes this issue. Now, `”` is supported as a valid default property within the parameters section. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[*BZ#1966275*])
+
+* Previously, the Cluster Version Operator reported a `ClusterOperatorDegraded` alert when ClusterOperator resources were degraded for ten minutes, and this alert sometimes fired prematurely during installation as resources were still being created. This update changes the 10-minute period to 30 minutes, providing enough time for the installation to progress without premature `ClusterOperatorDegraded` alerts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1957991[*BZ#1957991*])
```

Comment:
Markup `ClusterOperator`.

Reply:
s/ten/10/ 

to match the use of numerals in the next sentence.

IBM SG: "You can choose whether to use numerals or words for numbers that are less than 10."

Reply:
Agreed! Changed!